docs/paper/admin/how-to/basic-troubleshooting.md

```diff
@@ -0,0 +1,143 @@
+---
+slug: /basic-troubleshooting
+---
+
+# Basic Troubleshooting
+
+This guide will help you diagnose your server's problem before reporting it to PaperMC or the plugin's author.
+
+:::caution Stop Your Server And Take A Backup
+
+Before following this guide, stop your server first. Modifying server files while it is still running will corrupt them.  
+Only a full server shutdown can prevent this.
+
+Also, if you don't follow this guide carefully or make a mistake while following this guide, you might corrupt your server. It is highly advised to backup your server before following this guide.  
+It would be ideal to create a test server by copying your production server's file, but that's not always possible.
+
+Please, take a backup first.
```

Comment:
Remove this line

Reply:
It is just repetitive

Reply:
The heading and text already tell you to create a backup two times, that last line is unnecessary

Reply:
If I remove whole line according to the suggestion, things would be like:

Heading: Stop and backup your server  
Text: Stop your server

So two times of 'stop your server' and only one 'backup your server'. It looks like something about 'backup your server' is missing in text body. So there should be at least one line about saying 'backup your server'.

Reply:
Oh, I misunderstood how GitHub's review looks like. So Kenny was taking about the line `Please, take a backup first.`, right?